lib/matplotlib/cbook/__init__.py

```diff
@@ -2337,3 +2338,16 @@ def _backend_module_name(name):
     """
     return (name[9:] if name.startswith("module://")
             else "matplotlib.backends.backend_{}".format(name.lower()))
+
+
+def _without_gc(f):
```

Comment:
Docstring: """A decorator to disable garbage collection in the decorated function."""